MongoDB 6.0 User Guide (#628)
mulesoft/docs-connectors: modules/ROOT/pages/mongodb/mongodb-connector-6-0.adoc

> = MongoDB Connector 6.0 - Mule 4

Reviewer: HERE

ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]

MongoDB Connector Version 6.0

Anypoint Connector for MongoDB (MongoDB Connector) is a closed-source connector that provides a connection between the Mule runtime engine (Mule) and third-party software on a MongoDB server.

Release Notes: xref:release-notes::connector/mongodb-connector-release-notes-mule-4.adoc[MongoDB Connector Release Notes] +
Exchange: https://www.mulesoft.com/exchange/com.mulesoft.connectors/mule-mongodb-connector/[MongoDB Connector]

== Prerequisites

To use this information, you must be familiar with MongoDB, Mule runtime engine (Mule), Anypoint Connectors, Anypoint Studio, elements in a Mule flow, and global elements.

> Before creating an app, you must have access to the MongoDB target resource and Anypoint Platform. You must also understand how to create a Mule app using Design Center or Anypoint Studio.

Reviewer: ```suggestion
Before creating an app, you must have access to the MongoDB target resource and Anypoint Platform. You must also understand how to create a Mule app using Anypoint Design Center or Anypoint Studio.
```

> To use the this connector you need:

Reviewer: ```suggestion
To use this connector, you need:
```

* A MongoDB instance
* A set of valid credentials, including the required MongoDB endpoints that point to your instance

== Audience

> * Starting user -- To create your Mule app, see

Reviewer: ```suggestion
* Starting user 
To create your Mule app, see
```

xref:mongodb/mongodb-connector-6-0-design-center.adoc[Design Center] or
xref:mongodb/mongodb-connector-6-0-studio.adoc[Anypoint Studio].

> * Power user - Read xref:mongodb/mongodb-connector-6-0-xml-maven.adoc[XML and Maven Support]

Reviewer: ```suggestion
* Power user
Read xref:mongodb/mongodb-connector-6-0-xml-maven.adoc[XML and Maven Support]
```

and xref:mongodb/mongodb-connector-6-0-examples.adoc[Examples].

== Exchange Example

> https://www.mulesoft.com/exchange/org.mule.examples/importing-a-CSV-file-into-Mongo-DB/[Anypoint Exchange] has an example that shows how to import data in a CSV format from a local directory into MongoDB.

Reviewer: Exchange has an example?
```suggestion
https://www.mulesoft.com/exchange/org.mule.examples/importing-a-CSV-file-into-Mongo-DB/[Anypoint Exchange] documentation includes an example that shows how to import data in a CSV format from a local directory into MongoDB.
```

Author: There is an example in Exchange.

== Next Steps

> After you have met the prerequisites, and experimented with example, you are ready to create an app with xref:mongodb/mongodb-connector-6-0-design-center.adoc[Design Center] or

Reviewer: ```suggestion
After you have met the prerequisites and experimented with the example, you are ready to create an app with xref:mongodb/mongodb-connector-6-0-design-center.adoc[Design Center] or
```

Release Notes: xref:release-notes::connector/mongodb-connector-release-notes-mule-4.adoc[MongoDB Connector Release Notes] +
Exchange: https://www.mulesoft.com/exchange/com.mulesoft.connectors/mule-mongodb-connector/[MongoDB Connector]

== Prerequisites

To use this information, you must be familiar with MongoDB, Mule runtime engine (Mule), Anypoint Connectors, Anypoint Studio, elements in a Mule flow, and global elements.

Before creating an app, you must have access to the MongoDB target resource and Anypoint Platform. You must also understand how to create a Mule app using Design Center or Anypoint Studio.

To use the this connector you need:

* A MongoDB instance
* A set of valid credentials, including the required MongoDB endpoints that point to your instance.

== Audience

* Starting user -- To create your Mule app, see
xref:mongodb/mongodb-connector-6-0-design-center.adoc[Design Center] or
xref:mongodb/mongodb-connector-6-0-studio.adoc[Anypoint Studio].
* Power user - Read xref:mongodb/mongodb-connector-6-0-xml-maven.adoc[XML and Maven Support]
and xref:mongodb/mongodb-connector-6-0-examples.adoc[Examples].

== Exchange Example

> https://www.mulesoft.com/exchange/org.mule.examples/importing-a-CSV-file-into-Mongo-DB/[Anypoint Exchange] has an example that shows how to import data in a CSV format from a local directory into MongoDB. *Reviewers, is this example going to be updated for the 6.0 release, or will there be a new example?*

Reviewer 2: @gastonbritomulesoft  will try to update the example
